### ISSUE TITLE
[CKEditor 5] How can I custom serviceProviderProperties for npm wiris/mathtype-ckeditor5 installation

### ISSUE THREAD

**Issue by LinhTT46** [wiris/html-integrations#851]
Hello, 

I used MathType in CKEditor 5 by npm installation. This plugin is very cool. 
However, I have a problem when customizing the serviceProviderProperties, change wiriseditorparselatex to false.
The document guides PHP and Java only. 
![Screenshot from 2023-10-05 22-10-21](https://github.com/wiris/html-integrations/assets/44936298/4fa59cca-bb38-42db-90ad-dd0117f84663)
I don't know how to customize the configuration above for the npm installation. 
please tell me how can I customize the configuration file when integrating MathType by npm installation. Thanks!

**Comment by deveshparmar** [comment]
Yeah I'm also encountering same issue with tinymce editor cant able to provide configurations to tinymce
